platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/isolated/IsolatedProjectsAccessFromGroovyDslIntegrationTest.groovy

```diff
@@ -68,12 +68,12 @@ class IsolatedProjectsAccessFromGroovyDslIntegrationTest extends AbstractIsolate
         then:
         fixture.assertStateStoredAndDiscarded {
             projectsConfigured(":", ":a", ":b")
-            problem("Build file 'build.gradle': line 3: Cannot access project ':a' from project ':'")
-            problem("Build file 'build.gradle': line 3: Cannot access project ':b' from project ':'")
-            problem("Build file 'build.gradle': line 4: Cannot access project ':a' from project ':'")
-            problem("Build file 'build.gradle': line 4: Cannot access project ':b' from project ':'")
-            problem("Build file 'build.gradle': line 5: Cannot access project ':a' from project ':'")
-            problem("Build file 'build.gradle': line 5: Cannot access project ':b' from project ':'")
+            problem("Build file 'build.gradle': line 3: Cannot access 'plugins' on project ':a' from project ':'")
+            problem("Build file 'build.gradle': line 3: Cannot access 'plugins' on project ':b' from project ':'")
+            problem("Build file 'build.gradle': line 4: Cannot access 'java' on project ':a' from project ':'")
```

Comment:
💅 Could we say `'java extension'` here instead?

Reply:
Done